packages/runtime/src/schema.ts

```diff
@@ -166,12 +171,16 @@ const valueFilterInputs = {
   FloatValueFilter: createScalarValueFilterInput(GraphQLFloat),
   IntValueFilter: createScalarValueFilterInput(GraphQLInt),
   StringValueFilter: createScalarValueFilterInput(GraphQLString),
+  DateValueFilter: createScalarValueFilterInput(GraphQLDate),
+  DateTimeValueFilter: createScalarValueFilterInput(GraphQLDateTime),
```

Comment:
These shouldn't be needed, scalars that extend from `String` can simply use the `StringValueFilter`

Reply:
@dbcfd any advantage using specific types here?

Reply:
@PaulLeCam followed up with Danny around this as well. It seems using those types makes query introspection work better with other clients e.g. Apollo which would come in handy because I think thats what devs are using for actual app implementation (React) and then responds raise errors correctly when non dates are being passed in. If you don't see any major issue otherwise I would vote to keep them as is.